app/controllers/sakes_controller.rb

```diff
@@ -162,6 +172,23 @@ def delete_photos
       photo.destroy if params[photo.chackbox_name] == "delete"
     end
   end
+
+  # update後のリダイレクト処理
+  # 編集画面からupdateする場合: 詳細ページにリダイレクト
+  # HTTP_REFERERが設定されていない場合: 詳細ページにリダイレクト
+  # 上記のどちらにも当てはまらない場合: ユーザーが直前にいたページにリダイレクト
```

Comment:
YARDはアブスト、詳細を区切ると判定してくれるよ
```suggestion
  # update後のリダイレクト処理
  #
  # 編集画面からupdateする場合: 詳細ページにリダイレクト
  # HTTP_REFERERが設定されていない場合: 詳細ページにリダイレクト
  # 上記のどちらにも当てはまらない場合: ユーザーが直前にいたページにリダイレクト
```

Reply:
あとコロン使うよりも文章を書いたほうがいい。

```ruby
# ○○の場合は△△される。
# ××の場合は□□される
```

でいいと思う

Reply:
Fixed at 1b289e470e461c311efc009ec14364ee87d63734